tests/test_urlparse.py

```diff
@@ -529,17 +526,15 @@ def _encode(t):
             self.assertEqual(result.url, defrag)
             self.assertEqual(result.fragment, frag)
 
-    @pytest.mark.xfail
     def test_urlsplit_scoped_IPv6(self):
         p = urlparse4.urlsplit('http://[FE80::822a:a8ff:fe49:470c%tESt]:1234')
-        self.assertEqual(p.hostname, "fe80::822a:a8ff:fe49:470c%tESt")
+        self.assertEqual(p.hostname, "fe80::822a:a8ff:fe49:470c%test")
```

Comment:
why is it lowercased?

Reply:
I tried to test this on urllib urlsplit and it gave the result just exactly the way I changed it. So I thought it might be a good idea to modify this test.
```
>>> from urllib.parse import urlsplit
>>> urlsplit('http://[FE80::822a:a8ff:fe49:470c%tESt]:1234').hostname
'fe80::822a:a8ff:fe49:470c%test'
```

Reply:
I see. And did the implementation change? I mean, it was passing without lowercasing before, and now it's passing again, right?

Reply:
I think this is an error in the new implementation of urllib. I think it's mentioned [here](https://github.com/python/cpython/blob/master/Lib/urllib/parse.py#L160)... let me change this back

Reply:
I also put this test back to normal! I'm sorry for merging this too early....